lib/controller/BibliotheekController.class.php

```diff
@@ -40,7 +48,27 @@ public function __construct($query) {
 	}
 
 	public function performAction(array $args = array()) {
-		parent::performAction($args);
+		if (in_array($this->action, ['boek', 'import', 'addexemplaar'])) {
```

Comment:
Deze logica is erg ingewikkeld, je kan niet makkelijk zien hoe de routes werken. Kan misschien in een vervolg pr wat verbeterd worden.